pkg/chart/gcp.go

```diff
@@ -222,6 +226,66 @@ func createCloudSQLInstance(ctx context.Context, teamSlug, dbInstance, gcpProjec
 	return nil
 }
 
+type instanceOperation struct {
+	Name string `json:"name"`
+}
+
+func awaitInstanceOperationsComplete(ctx context.Context, dbInstance, gcpProject string) error {
+	runningOps, err := listRunnningSQLInstanceOperations(ctx, dbInstance, gcpProject)
+	if err != nil {
+		return err
+	}
+
+	for _, op := range runningOps {
+		cmd := exec.CommandContext(ctx,
```

Comment:
Kul løsning! Hva tenker du om å slenge på en eksplisitt `--timeout=300`? Ser fra docen du linke over at den defaulter til dette: ```--timeout=TIMEOUT; default=300```. Liker bare tanken på at man kan se fra kommandoen hvor lenge den vil vente 🤷🏽

Reply:
God idé

Reply:
Er det en ok løsning å sette tiden til context deadline som timeout for kommandoen, og defaulte til 300 sekunder dersom konteksten ikke har timeout?

Reply:
høres bra ut!